indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -22,18 +22,21 @@
 from indico.modules.events.abstracts.controllers.management import RHManageAbstractsBase
 from indico.modules.events.abstracts.forms import (EditEmailTemplateRuleForm, EditEmailTemplateTextForm,
                                                    CreateEmailTemplateForm)
+from indico.modules.events.abstracts.models.abstracts import AbstractState
 from indico.modules.events.abstracts.models.email_templates import AbstractEmailTemplate
 from indico.modules.events.abstracts.notifications import get_abstract_notification_tpl_module
 from indico.modules.events.abstracts.util import build_default_email_template, create_mock_abstract
 from indico.web.flask.templating import get_template_module
 from indico.web.util import jsonify_data, jsonify_template
+from werkzeug.exceptions import BadRequest
```

Comment:
This is thirdparty, not indico so it should be in the previous block of imports (below the flask one)